GAE/src/org/waterforpeople/mapping/dao/SurveyInstanceDAO.java

```diff
@@ -629,13 +629,14 @@ public void deleteSurveyInstance(SurveyInstance surveyInstance) {
         QuestionAnswerStoreDao qasDao = new QuestionAnswerStoreDao();
         QuestionDao qDao = new QuestionDao();
         List<QuestionAnswerStore> qasList = qasDao.listBySurveyInstance(surveyInstanceId);
+        SurveyQuestionSummaryDao summDao = new SurveyQuestionSummaryDao();
```

Comment:
I thought this was already initialised with the `@Inject` statement at the top of the file?